apps/etterlatte-fordeler/src/main/kotlin/fordeler/FordelerService.kt

```diff
@@ -69,7 +71,7 @@ class FordelerService(
                     " Se sikkerlogg for detaljer",
             )
             sikkerLogg.info("Fikk en søknad med en familierelasjon som manglet ident", e)
-            IkkeGyldigForBehandling(listOf(FordelerKriterie.FAMILIERELASJON_MANGLER_IDENT))
+            FordelerResultat.TrengerManuellJournalfoering(FordelerKriterie.FAMILIERELASJON_MANGLER_IDENT.forklaring)
```

Comment:
Denne sjekken må nok ta høyde for featureToggle'en for åpning av kriterier i produksjon